typegate/src/typegraph/mod.ts

```diff
@@ -349,6 +351,23 @@ export class TypeGraph {
     );
   }
 
+  getRandom(
+    schema: TypeNode,
+  ): number | string | null {
+    const tgTypes: TypeNode[] = this.tg.types;
+    const chance: typeof Chance = new Chance(1);
```

Comment:
The seed should be configurable on the typegraph...
With something like:
```python
g.configureRandomInjection(seed=1)
```
It is mostly needed for the tests to be deterministic, but I think it should be random by default.

Reply:
Ok, I will work on doing that.

Reply:
@Natoandro, I was trying to provide a gateway to configure the seed from the typegraph as per your suggestion like this
```python 
g.configureRandomInjection(seed=1)
```
The similar implementation that I found to what I am trying to achieve is `g.inherit()` which calls the `InheritDef()` class. But I wasn't able to extract as much as info I need to add the method to configure the seed for random injection as 'g.inherit()' is used in a different use case. Are there any similar implementations that have the same use case(adding configs from the typegraph) as `g.configureRandomInjection(seed=1)` that I can look into?

Reply:
@destifo you need to save the seed before finalizing the typegraph so maybe you can use a store similarly this value
https://github.com/metatypedev/metatype/blob/main/typegraph/core/src/global_store.rs#L63

Reply:
oww ok, so I should be saving the configured seed value inside the store as a field @afmika? just like `deploy_cwd_dir`?

Reply:
I believe you could also use g.seed(0) which might be used by different runtimes as well

Reply:
Ok @zifeo, I will look into the details and apply your suggestions.

Reply:
Done.